openstack/compute/v2/extensions/resetstate/results.go

```diff
@@ -0,0 +1,11 @@
+package resetstate
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// ResetResult is the response from a ResetState operation. Call its ExtractErr
```

Comment:
is the response of a

Reply:
@jamiehannaford Thanks for review fixed super minor nits.